src/App.vue

```diff
@@ -3,6 +3,15 @@
     <navigation-drawer />
     <v-main>
       <router-view />
+
+      <!-- temporary locale changer -->
+      <div class="locale-changer">
+        <select v-model="$i18n.locale">
+          <option v-for="(lang, i) in $i18n.availableLocales" :key="`${i}-${lang}`" :value="lang">
```

Comment:
Can loop available locales instead of hardcoding them.

Reply:
We'll also need a better place for this to live, and maybe some way to set the initial language based on the user's locale. That last one can be a follow up, though.

Reply:
That makes sense. I'll leave this comment open until we write up the issue so we remember to doc it